src/gluonts/model/gpvar/_estimator.py

```diff
@@ -379,7 +379,6 @@ def create_training_network(self) -> GPVARTrainingNetwork:
             lags_seq=self.lags_seq,
             scaling=self.scaling,
             distr_output=self.distr_output,
-            conditioning_length=self.conditioning_length,
```

Comment:
Removing `conditioning_length` from network since it is not used there. @mbohlkeschneider please confirm this.

Reply:
It is not used in the network but rather in the transformation. So it can be removed here but maybe should still be in `DeepVAR` to be able to use the cdf transform.